bluebrain/repo-bluebrain/packages/py-bbp-workflow/package.py

```diff
@@ -39,6 +39,8 @@ class PyBbpWorkflow(PythonPackage):
     depends_on("py-cwl-luigi", type="run")
     depends_on("py-cwl-registry", type="run")
     depends_on("py-brayns", type="run")
+    depends_on("py-bluepyemodel", type="run")
+    depends_on("py-bluepyemodelnexus", type="run")
```

Comment:
Actually, are these defined in `setup.py`? Upstream canonically seems to want `"build", "run"` for everything in `setup.py`, a habit we'd like to adopt, too.

Reply:
yep, they are in `setup.py`

Reply:
I'll add `"build", "run"` with the next bump.